lizard_efcis/import_data.py

```diff
@@ -709,11 +719,12 @@ def manual_import_csv(self, filename, mapping_code,
                         continue
                     else:
                         logger.error(ex.message)
-                        action_log.update({"IntegrityError%s" % count: ex.message})
+                        action_log.update(
+                            {"IntegrityError%s" % count: ex.message})
                         break
                 except Exception as ex:
                     action_log.update({"Error%s" % count: ex.message})
                     break
-                is_imported = True
+            is_imported = True
```

Comment:
Even nakijken of de `is_imported = True` op het juiste indentatie-niveau staat. Ik kan het zo in de diff niet zien.